engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/function/ApplicationOperator.java

```diff
@@ -12,7 +12,8 @@
 @BuiltinMethod(
     type = "Function",
     name = "<|",
-    description = "Takes a function and an argument and applies the function to the argument.")
+    description = "Takes a function and an argument and applies the function to the argument.",
+    needsFrame = false)
```

Comment:
This is, again, confusing - a node that has `VirtualFrame` parameter in its `execute` method and at the same time specifies `needsFrame = false` in the annotation. I would prefer the name `forceInlining` for the annotation.

Reply:
OK, let's change `needsFrame` to something else. Let the [discussion happen there](https://github.com/enso-org/enso/pull/6442#discussion_r1179013345).